code/src/pages/home/components/call-to-action-section/CallToActionSection.tsx

```diff
@@ -0,0 +1,67 @@
+import AppContainer from "@/components/app-container/AppContainer";
+import AppBox from "@/components/app-box/AppBox";
+import AppTypography from "@/components/app-typography/AppTypography";
+import AppButton from "@/components/app-button/AppButton";
+import AppLink from "@/components/app-link/AppLink";
+
+import callToActionItems from "@/pages/home/components/call-to-action-section/CallToAction.constants";
+
+import "@/pages/home/components/call-to-action-section/CallToActionSection.scss";
+
+type CallToActionSectionItemProps = {
+  titleTranslationKey: string;
+  descriptionTranslationKey: string;
+  captionTranslationKey: string;
+  linkTo: string;
+  imageUrl: string;
+  buttonLabelTranslationKey: string;
+};
+
+const CallToActionSectionItem = ({
```

Comment:
move it to separate component

Reply:
Moved. Please check if i got you correctly

Reply:
Yeah, that's it